### PR TITLE
Update maintenance list

### PR DESCRIPTION
**Pull request by karareinsel**
Updating the maintenance list on cloud.gov to reflect current practices.

### CODE REVIEW

File: content/docs/ops/maintenance-list.md [hunk 1]
```diff
@@ -12,24 +12,16 @@ personally, you are responsible for ensuring that an appropriate person does it.
...
-- Meet with the previous support person and take responsibility for any open
-support items they are still working on. There is a standing support sync meeting
-(Tuesday 4:30-5 pm ET) that you should join if you are rolling on or off support
-that week.
 
-## At least once during your week of support
 
+## Weekly support tasks
+
+- Update the #cg-support topic to include your name as the support contact.
```
Comment (bengerman13) [inline content/docs/ops/maintenance-list.md]: I think we should add this back to the top heading, and revert back to "is this your first day of support"

Comment (bengerman13) [inline content/docs/ops/maintenance-list.md]: it might be nice to have`#cg-support` be a link

File: content/docs/ops/maintenance-list.md [hunk 2]
```diff
@@ -91,12 +83,7 @@ deploying an update that will upgrade the stemcell. You should also read the
 
 ## Review and respond to open alerts
 
-Review all recent alerts and notifications delivered to [`cg-notifications`](https://groups.google.com/a/gsa.gov/forum/#!forum/cloud-gov-notifications)
-, [`#cg-platform-news`](https://gsa-tts.slack.com/messages/cg-platform-news/)
-
-### Are there no alerts or notifications?
-Verify the monitoring system is functioning correctly and confirm that alerts
-are reaching their expected destinations.
+Review active alerts at https://prometheus.fr.cloud.gov/alerts and review all production smoke tests to ensure they are passing.
```
Comment (bengerman13) [inline content/docs/ops/maintenance-list.md]: maybe split this into two bullet points

File: content/docs/ops/maintenance-list.md (hunk 1 above)
Comment (karareinsel) [reply content/docs/ops/maintenance-list.md]: I'm fine reverting back to the original header name (Is this your first day of support).

**Review by bengerman13 — APPROVED** [review]
LGTM

**Pull request merged**